### PR TITLE
loading-squares entry (#2)

### PR DESCRIPTION
**Pull request by Morphyish**
This SVG is a loading animation made of 10 rotating squares.

I had to use inline CSS variables to make it under the 1Ko limit, which might be an issue on some browsers (looking at you Edge).

On the flip side I'm using the currentColor property, making it easier to theme.

### CODE REVIEW

File: img/loading-squares.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 150 150"><style>.c{transform-origin:50px 50px;transform:rotate(0);animation:c 4s infinite}@keyframes c{0%{transform:rotate(0)}90%{transform:rotate(0)}100%{transform:rotate(90deg)}}rect{width:30px;height:30px;opacity:.3;transform-origin:var(--o);fill:currentColor;animation:b 4s infinite var(--d);}@keyframes b{0%{transform:rotate(0);opacity:.3}5%{opacity:1}10%,100% {transform:rotate(90deg);opacity:.3}}</style><g transform="translate(25 25) rotate(45 50 50)"><g class="c"><rect x="0" y="0" style="--o:15px 15px;--d:0s"/><rect x="35" y="0" style="--o:50px 15px;--d:.4s"/><rect x="70" y="0" style="--o:85px 15px;--d:.8s"/><rect x="0" y="35" style="--o:15px 50px;--d:1.2s "/><rect x="35" y="35" style="--o:50px 50px;--d:1.6s "/><rect x="70" y="35" style="--o:85px 50px;--d:2s "/><rect x="0" y="70" style="--o:15px 85px;--d:2.4s "/><rect x="35" y="70" style="--o:50px 85px;--d:2.8s "/><rect x="70" y="70" style="--o:85px 85px;--d:3.2s "/></g></g></svg>
```
Comment (JeremiePat) [inline img/loading-squares.svg]: I think you could add a `svg{color:#???}` declaration in your `<style>` element in order to "show" the `currentColor` usage.

**Review by JeremiePat — CHANGES_REQUESTED** [review]

**Comment by JeremiePat** [comment]
Ah, that's awesome, I like it.

> I had to use inline CSS variables to make it under the 1Ko limit, which might be an issue on some browsers (looking at you Edge).

No, it's okay, CSS custom properties are supported by Edge since v15 and as it is now a chromium based browser it will work anyway. Only IE isn't supporting CSS custom properties, but we do not need to care about that dying browser here.

> On the flip side I'm using the currentColor property, making it easier to theme.

That's great! See my review comment for a little improvement on that feature.

File: img/loading-squares.svg (hunk 1 above)
Comment (Morphyish) [reply img/loading-squares.svg]: Done

**Review by JeremiePat — APPROVED** [review]

**Comment by JeremiePat** [comment]
Awesome thanks for the contribution which is live now 😁

**Pull request merged**